.github/workflows/integration_tests.yml

```diff
@@ -64,8 +64,21 @@ jobs:
           done
         shell: bash
 
+      - name: Submit to IonQ Simulator
```

Comment:
I'm not sure about this. @bettinaheim said to only run the syntax checker every night, not the simulators.

Reply:
For Quantinuum. We have unlimited access to the IonQ simulator, so running on it every night is fine.